wezterm-gui/src/termwindow/mod.rs

```diff
@@ -1756,17 +1757,17 @@ impl TermWindow {
         Ok(())
     }
 
-    fn scroll_by_page(&mut self, amount: isize) -> anyhow::Result<()> {
+    fn scroll_by_page(&mut self, amount: NotNan<f64>) -> anyhow::Result<()> {
```

Comment:
I think it looks a little less magical to pass `f64` directly here, and extract the value from the `NotNan` at the call site.

```suggestion
    fn scroll_by_page(&mut self, amount: f64) -> anyhow::Result<()> {
```